docs/02-app/02-api-reference/05-next-config-js/devIndicators.mdx

```diff
@@ -7,6 +7,54 @@ description: Optimized pages include an indicator to let you know if it's being
 
 <AppOnly>
 
+`devIndicators` is a `next.config.js` option that allows you to configure the on-screen indicators that give context about the current route during development.
+
+```ts filename="Types"
+  devIndicators: {
+    appIsrStatus?: boolean, // defaults to true
+    buildActivity?: boolean, // defaults to true
+    buildActivityPosition?: 'bottom-right'
+    | 'bottom-left'
+    | 'top-right'
+    | 'top-left', // defaults to 'bottom-right'
+  },
+```
+
+## `appIsrStatus` (Static Indicator)
+
+Next.js displays a static indicator in the bottom-left corner of the screen that signals if a route will be prerendered at build time. This makes it easier to understand whether a route is static or dynamic, and for you to identify if a route [opts out of static rendering](#static-route-not-showing-the-indicator).
```

Comment:
```suggestion
Next.js displays a static indicator in the bottom corner of the screen that signals if a route will be prerendered at build time. This makes it easier to understand whether a route is static or dynamic, and for you to identify if a route [opts out of static rendering](#static-route-not-showing-the-indicator).
```
Tbd on positioning, we're going to have it in the same corner as the dev overlay warning box I believe though.